pkg/services/folder/folderimpl/sqlstore_test.go

```diff
@@ -632,30 +756,35 @@ func CreateOrg(t *testing.T, db *sqlstore.SQLStore) int64 {
 	return orgID
 }
 
-func CreateSubTree(t *testing.T, store *sqlStore, orgID int64, parentUID string, depth int, prefix string) []string {
+func CreateSubTree(t *testing.T, store *sqlStore, service *Service, depth int, prefix string, cmd folder.CreateFolderCommand) []string {
```

Comment:
This function has become over-complicated: I would suggest leave this one as as is and introduce a similar that uses the service instead (or the dashboard folder store).